presto-elasticsearch/src/test/java/com/facebook/presto/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -42,63 +46,72 @@ private ElasticsearchQueryRunner() {}
     private static final String TPCH_SCHEMA = "tpch";
     private static final int NODE_COUNT = 2;
 
-    public static DistributedQueryRunner createElasticsearchQueryRunner(EmbeddedElasticsearchNode embeddedElasticsearchNode, Iterable<TpchTable<?>> tables)
+    public static DistributedQueryRunner createElasticsearchQueryRunner(
+            HostAndPort address,
+            Iterable<TpchTable<?>> tables,
+            Map<String, String> extraProperties,
+            Map<String, String> extraConnectorProperties)
             throws Exception
     {
+        RestHighLevelClient client = null;
         DistributedQueryRunner queryRunner = null;
         try {
             queryRunner = DistributedQueryRunner.builder(createSession())
-                    .setNodeCount(NODE_COUNT)
+                    .setExtraProperties(extraProperties)
```

Comment:
Why remove `setNodeCount`?

Reply:
Trino removed that and other counts in https://github.com/trinodb/trino/pull/3267. I should do it in another PR. Added it back. Thanks